src/js/controllers/tab-home.js

```diff
@@ -213,7 +213,10 @@ angular.module('copayApp.controllers').controller('tabHomeController',
         walletService.getStatus(wallet, {}, function(err, status) {
           if (err) {
 
-            wallet.error = (err === 'WALLET_NOT_REGISTERED') ? gettextCatalog.getString('Wallet not registered') : bwcError.msg(err);
+            // if wallet not registered, automtically try to recreate
+            if (err === 'WALLET_NOT_REGISTERED') {
```

Comment:
We need to whitelist hostnames for this to be more secure -- especially since this is configurable.

Reply:
Right now we're using bitcoinz.ph but we plan to use bws.btcz.rocks in the future. Maybe you can check against both hostnames for now.

Reply:
I don't know if I agree with only using bws.btcz.rocks. Maybe we need a bunch of servers here.

Reply:
hostnames aren't servers. One hostname can point to many servers, which is the way we've been and plan to continue running.

Reply:
Sorry, I meant hostnames not servers. I’m thinking the best thing to do here is randomly select a group of servers from a seed list. If one of those stops working then we can move to another group of servers.

Reply:
Err... hostnames

Reply:
That approach introduces a couple of things:
1. Time to query servers will cause a delay on app initialization -- especially if multiple servers are down. We may be able to test hostnames in a background thread while the app is initializing but we need to make sure we delay any process that makes requests until this is done.
2. Updates will require releasing a new app version. I'd prefer to manage the hostnames via the same predictable hostname, such as:
- bws-1.btcz.rocks
- bws-2.btcz.rocks
- bws-{n}.btcz.rocks

This allows us to put servers in, take servers out, etc. without having to force people to update to receive the new list of servers.

Reply:
1. Decentralization should be our main goal and using one domain name is not secure. We need multiple clusters of servers, or this is a centralized solution. Also, querying multiple servers is not a time consuming thing and there is a 'race' function with javascript promises that can actually find and use the fastest server to respond. 
2. I think a list of hostnames is fine for now. Eventually we will want to add a seeder that will return many servers.

Reply:
Sounds good. But we need some sort of protection before calling `recreateWallet`. I think calling it without any checks is dangerous.